### ISSUE TITLE
Inspect scaling bug

### ISSUE THREAD

**Issue by Toxicom** [Toxicom/toxiui#66]
### Description

```lua
15x Blizzard_InspectUI/InspectPVPFrame.lua:71: Usage: UnitFactionGroup("unit" [,checkDisplayRace])
[string "=[C]"]: in function `UnitFactionGroup'
[string "@Blizzard_InspectUI/InspectPVPFrame.lua"]:71: in function `InspectPVPFrame_Update'
[string "@Blizzard_InspectUI/InspectPVPFrame.lua"]:60: in function <Blizzard_InspectUI/InspectPVPFrame.lua:58>

Locals:
(*temporary) = nil
```

Basically the issue here is that we force load `Blizzard_InspectUI` addon so that we can scale the inspect frame.
A very similar issue described here: https://github.com/nickbock/ElvUI_Enhanced_Again/issues/94#issuecomment-476096729

### Notes

- https://github.com/Toxicom/toxiui/blob/development/Modules/Misc/AdditionalScaling.lua#L35

### To reproduce

1. Scale inspect frame in `/tx settings`
2. Reload
3. Hovering a player should pop the error

### Acceptance criteria

- [ ] Inspect Frame scaling works without force loading `Blizzard_InspectUI`

**Comment by Toxicom** [comment]
WindTools does something that we should do with their skinning for the inspect frame:
- https://github.com/fang2hou/ElvUI_WindTools/blob/development/Modules/Skins/Blizzard/Inspect.lua#L47
- https://github.com/fang2hou/ElvUI_WindTools/blob/development/Modules/Skins/Core.lua#L281

But I don't fully understand how it works yet

**Comment by Toxicom** [comment]
Perhaps this could be used somehow? https://github.com/Toxicom/toxiui/blob/development/Core/Functions/Event.lua#L273
Tried it like this but doesn't work (frame doesn't get scaled)
```lua
  F.Event.ContinueOnAddOnLoaded("Blizzard_InspectUI", function()
    M:SetElementScale(syncedFrameName, "InspectFrame")
  end)
```

**Comment by Toxicom** [comment]
Fixed with #71
